bin/notification.py

```diff
@@ -6,157 +6,31 @@
 
 import json
 import logging
-from botocore.vendored import requests
-from abc import ABC, abstractmethod
-
-import settings
-from model import AthenaQuery, AnomalyDetectionEvent, AnomalyDetectionMessage, AnomalyDetectionSns
+from typing import Sequence
 
+from . import settings
+from .model import UnknownEventException
+from .notificators.notificator import Notificator
 
 logger = logging.getLogger()
 
+
 def lambda_handler(event, context):
     """
-    This lambda handler may be invoked by more than one events:
-    as of Jan 2020 its hard threshold notification or anomaly detection notification.
-    We have a simple logic which recognizes which event do we have at hand and we invoke an appropriate notifier
+    This handles events and may handle more than one event type.
+    If you have more events that you'd like to be notified on,
+    register them in settings.py under NOTIFICATORS
     """
-    hard_threshold_notificator = HardThresholdNotificator(config=settings)
-    anomaly_detection_notificator = AnomalyDetectionNotificator(config=settings)
+    notificators: Sequence[Notificator] = [notificator(config=settings) for notificator in settings.NOTIFICATORS]
```

Comment:
Instead of modifying bin/notification.py to add more Notificators (previously hard_threshold_notificator, anomaly_detection_notificator), we import a setting `NOTIFICATORS`, a Sequence of Notificators to be used.